crates/flox/src/commands/mod.rs

```diff
@@ -469,7 +471,46 @@ impl EnvironmentSelect {
                             if activated == current_dir {
                                 Self::open_env_pointer(flox, &current_dir, current_dir_pointer)
                             } else {
-                                todo!("needs design");
+                                let activated_pointer = EnvironmentPointer::open(&activated)?;
+                                let message = "Do you want to install to the current directory's flox environment or the current active flox environment?";
+                                let current_description = hacky_environment_description(
+                                    &current_dir,
+                                    &current_dir_pointer,
+                                )?;
+                                let activated_description =
+                                    hacky_environment_description(&activated, &activated_pointer)?;
+                                if Dialog::can_prompt() {
+                                    let dialog = Dialog {
+                                        message,
+                                        help_message: None,
```

Comment:
@jennymahmoudi I think the default help_message is a bit better than `Use arrows to select:` as specified in the ticket
<img width="1006" alt="Screenshot 2023-12-05 at 1 59 35 PM" src="https://github.com/flox/flox/assets/40775676/528448e8-ec17-4b49-9a83-1d609f6dd24f">

Reply:
Just mentioning that, if we change it we have to change it across all uses of the dialog..

Reply:
👍 no way you would have known, thanks!